src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int randomIndex = new Random().nextInt(Colors.values().length);
+        return Colors.values()[randomIndex].name();
```

Comment:
you should add `.toLowerCase()` in order to get a better output

just `name()` will return you `WHITE` instead of `white`

```suggestion
        return Colors.values()[randomIndex].name().toLowerCase();
```